toolkit/recordreplay/ProcessRecordReplay.cpp

```diff
@@ -219,13 +220,41 @@ MOZ_EXPORT void RecordReplayInterface_Initialize(int* aArgc, char*** aArgv) {
   }
   MOZ_RELEASE_ASSERT(dispatchAddress.isSome());
 
+  Maybe<std::string> apiKey;
+  {
+#ifdef _WIN32
```

Comment:
getenv and setenv are C library functions and should be defined on windows as well (maybe needs a `#define getenv _getenv` or something).  Either way though I think we can just remove this windows specific code since we're not yet in a place where we can test it.